boards/arm/mec1501modular_assy6885/mec1501modular_assy6885.dts

```diff
@@ -15,7 +15,7 @@
 
 	chosen {
 		zephyr,sram = &sram0;
-		zephyr,console = &uart1;
+		zephyr,console = &uart0;
```

Comment:
@scottwcpg 
Are you recommending to use the Pegasus header (J5) for UART instead the serial-to-USB port?  
Documentation for modular card has to be updated in such case

Reply:
My mistake. Having to switch between perforce and git all the time, I forget git pushes branch not individual commits.
I will try to rebase and remove the commit that changed the board from UART1 to UART0.  I used UART0 because I'm always using the same USB to UART cable and moving to different boards.

Reply:
Pushed a rebase to remove board changes.